docs/examples/rules.kts

```diff
@@ -204,5 +204,5 @@ val ruleSet = ruleSet(ortResult) {
 }
 
 
-// Populate the list of policy rule violation errors to return
+// Populate the list of policy rule violations to return.
```

Comment:
Shall we strip `policy` for consistency?

Reply:
I would agree, but I think @tsteenbe introduced it on purpose, so we should discuss it separately.